app/models/cases.py

```diff
@@ -24,7 +23,23 @@ class Case(BaseCase, TableModelMixin, table=True):
     )
 
 
-class CaseRequest(BaseCase):
-    """Request model used to create a new case"""
+class CaseRequest(BaseRequest):
+    case_type: CaseTypes
+    notes: List[CaseNotesRequest] | None
+    people: List[PersonRequest] | None
+    case_tracker: CaseTrackerRequest | None
+    eligibility_outcomes: List[EligibilityOutcomesRequest] | None
+
+    class Meta(BaseRequest.Meta):
+        related_fields = ["notes", "people", "case_tracker", "eligibility_outcomes"]
```

Comment:
Is there value in approaching this by iterating across all the fields of `Meta.model` and then link them if they are a Relationship?

Reply:
I think this can be doing using [SQLAlchemy inspect](https://docs.sqlalchemy.org/en/20/core/inspection.html):

For example
```python
from sqlalchemy.inspection import inspect 

  @property
  def related_fields(self) -> List[str]:
      if not self.Meta.model:
          return []
      return inspect(self.Meta.model).relationships.items()
```

Reply:
Thank you @BenMillar-MOJ , that's a really nice and simple way of discovering the related fields.

I updated the PR to reflect this